meta-mender-core/recipes-mender/mender-artifact/mender-artifact_3.9.0.bb

```diff
@@ -0,0 +1,32 @@
+require mender-artifact.inc
+
+################################################################################
+#-------------------------------------------------------------------------------
+# THINGS TO CONSIDER FOR EACH RELEASE:
+# - SRC_URI (particularly "branch")
+# - SRCREV
+# - DEFAULT_PREFERENCE
+#-------------------------------------------------------------------------------
+
+SRC_URI = "git://github.com/mendersoftware/mender-artifact.git;protocol=https;branch=3.9.x"
+
+# Tag: 3.9.0
+SRCREV = "fde2173c2e6969f516f33868a898700662d08746"
+
+# Enable this in Betas, and in branches that cannot carry this major version as
+# default.
+# Downprioritize this recipe in version selections.
+#DEFAULT_PREFERENCE = "-1"
+
+################################################################################
+
+# DO NOT change the checksum here without make sure that ALL licenses (including
+# dependencies) are included in the LICENSE variable below. Note that for
+# releases, we must check the LIC_FILES_CHKSUM.sha256 file, not the LICENSE
+# file.
+LICENSE = "Apache-2.0 & BSD-2-Clause & BSD-3-Clause & ISC & MIT & OpenSSL"
+
+LIC_FILES_CHKSUM = "file://src/github.com/mendersoftware/mender-artifact/LIC_FILES_CHKSUM.sha256;md5=ea777015e6e9552d5e57b38df7e68e07"
```

Comment:
Reminder: This is correct, but only for dunfell. For master and kirkstone you will need [the full listing](https://github.com/mendersoftware/meta-mender/blob/master/meta-mender-core/recipes-mender/mender-client/mender-client_3.3.0.bb#L27-L52), both for this and the ones below. [This tool](https://github.com/mendersoftware/mendertesting/blob/master/utils/make_bitbake_license_list.sh) can help you.

Reply:
Ah, you've already done it! Sorry, should have checked first! 😃